### PR TITLE
Light Backends and Brokers Guides - PR from issue #6539

### PR DESCRIPTION
**Pull request by danthegoodman1**
PR work for Issue: https://github.com/celery/celery/issues/6539

This adds some insight into when one might choose you use certain backends and brokers. This is not complete but the beginning of a framework for describing the advantages and disadvantages of each option.

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -213,6 +213,60 @@ database connections at ``fork``.
...
+`See broker documentation for details`_
+
+.. _`See broker documentation for details`: https://docs.celeryproject.org/en/stable/getting-started/brokers/redis.html
+
+**As a Backend:** Redis is a super fast K/V store, making it very efficient for fetching the results of a task call. As with the design of Redis, you do have to consider the limit memory available to store your data, and how you handle data persistence. If result persistence is important, consider using another DB for your backend.
+
+RabbitMQ
+--------
+
+RabbitMQ is a broker.
```
Comment (matusvalo) [inline README.rst:239]: I am missing here RabbitMQ as a backend usage using rpc:// backend

**Review by matusvalo — CHANGES_REQUESTED** [review]
I prefer to have this kind of information in documentation instead of README.rst - e.g. in following file:

 celery/docs/getting-started/brokers/index.rst

File: README.rst (hunk 1 above)
Comment (danthegoodman1) [reply README.rst]: That goes outside my experience with Celery, I'll look into learning more about it though.

**Comment by matusvalo** [comment]
@danthegoodman1 you moved it to `docs/getting-started/index.rst`  instead of `celery/docs/getting-started/brokers/index.rst`. Me personal preference is to have it in brokers part of documentation instead of general one.

**Comment by danthegoodman1** [comment]
> @danthegoodman1 you moved it to `docs/getting-started/index.rst` instead of `celery/docs/getting-started/brokers/index.rst`. My personal preference is to have it in brokers part of documentation instead of general one.

@matusvalo I had assumed we might want to keep it out of the brokers folder since my changes also include lots of backend suggestions. Should I consider result stores brokers?

**Comment by matusvalo** [comment]
Hmm good point. Let me review it once again :-)

**Comment by matusvalo** [comment]
I have went over the documentation once again and still I prefer to have this information present in Celery 5.0.5 documentation » Getting Started » Brokers section. Maybe we can rename the whole section to Brokers and Backends.

**Comment by danthegoodman1** [comment]
Alright so I changed the name of the folder from `brokers` to `backends-and-brokers` and moved it into the `index.rst` there, along with a few language changes for modified page to make sense.

**Review by matusvalo — APPROVED** [review]

**Comment by matusvalo** [comment]
Thank you @danthegoodman1. I merged your PR. I am still missing mentioning and explaining rpc backend but it can be done in another PR. If you have time any new PRs are welcome.

**Comment by danthegoodman1** [comment]
> Thank you @danthegoodman1. I merged your PR. I am still missing mentioning and explaining rpc backend but it can be done in another PR. If you have time any new PRs are welcome.

Thanks! I’ll definitely add more backends and brokers as I expand my experience with them.

**Comment by matusvalo** [comment]
BTW I have created PR with rpc:// backend mentioned: #6568 . Feel free to review.

**Pull request merged**